package.json

```diff
@@ -168,7 +168,6 @@
     "ts-node": "~8.3.0",
     "typescript": "~4.2.4",
     "webpack-cli": "^3.3.12",
-    "tslint": "~6.1.0",
```

Comment:
Are we installing `eslint` anywhere? Check Spartacus package.json. We might need all of the additiona `eslint-plugin` packages as well.

Reply:
I already did it.